api/prisma/seed.ts

```diff
@@ -110,6 +110,14 @@ export const initialDevSeed = async (): Promise<void> => {
                 .promise();
             console.log('Bucket created');
         }
+
+        try {
+            await sqs.getQueue('science-octopus-pdf-queue-local');
```

Comment:
The queue name is going to be 'science-octopus-pdf-queue-local' for all the stages?

Reply:
This is for local development

Reply:
Well, since you create the queue with a prefix + stage [here](https://github.com/JiscSD/octopus/pull/386/files#:~:text=QueueName%3A%20%60science%2Doctopus%2Dpdf%2Dqueue%2D%24%7Bprocess.env.STAGE%7D%60%2C)

Then you should get that queue back by using the same prefix + stage.
Hardcoding this will also break things if we run 'seed' command on other environments than local. (like int for example)

Reply:
We won't wont this on int, we have different seen data for that environment, plus this is out of scope, the local development has been pushed into another ticket for investigation and currently this works.

Reply:
There's no guarantee we will never run this script on int or any other environments. Why not simply updating the 'getQueue' method since it's bad practice to hardcode strings like that? 'createQueue' was perfectly fine as it was.

Reply:
It's largely irrelevant what whether it's hardcoded or not, since even if we did run it on another environment, the seed script would error as it needs the docker containers running, and if we did that we would use the local env vars anyway? no? but for the sake of getting this approved, I will change it.